### PR TITLE
Rearranged book 1 diffuse chapter to better motivate gamma

### PR DESCRIPTION
**Pull request by trevordblack**

### CODE REVIEW

File: books/RayTracingInOneWeekend.html [hunk 1]
```diff
@@ -2405,8 +2342,8 @@
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
             if (world.hit(r, interval(0.001, infinity), rec)) {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++
-                point3 target = rec.p + rec.normal + random_in_unit_sphere();
-                return 0.5 * ray_color(ray(rec.p, target - rec.p), depth-1, world);
+                vec3 direction = random_on_hemisphere(rec.normal);
+                return 0.5 * ray_color(ray(rec.p, direction), depth-1 world);
```
Comment (hollasch) [inline books/RayTracingInOneWeekend.html:2346]: Lost comma after `depth-1` parameter.

File: books/RayTracingInOneWeekend.html [hunk 2]
```diff
@@ -2510,6 +2447,107 @@
...
+whereas images that are transformed are said to be in _gamma space_. It is likely that the image
+viewer you are using is expecting an image in gamma space, but we are giving it an image in linear
+space. This is the reason why our image appears inaccurately dark.
+
+There are many good reasons for why images should be stored in gamma space, but for our purposes we
+just need to be aware of it. We are going to transform our data into gamma space so that our image
+viewer can more accurately display our image. As a simple approximation, we can use “gamma 2” as our
+transform, which is the power that you use when going from gamma space to linear space. We need to
+go from linear space to gamma space, which means taking the inverse of "gamma 2", so the power
+$1/\mathit{gamma}$, which is just the square-root.
```
Comment (hollasch) [inline books/RayTracingInOneWeekend.html:2506]: "so the power" → "which means an exponent of"

File: books/RayTracingInOneWeekend.html [hunk 3]
```diff
@@ -2510,6 +2447,107 @@
...
+        out << static_cast<int>(256 * intensity.clamp(r)) << ' '
+            << static_cast<int>(256 * intensity.clamp(g)) << ' '
+            << static_cast<int>(256 * intensity.clamp(b)) << '\n';
+    }
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    [Listing [write-color-gamma]: <kbd>[color.h]</kbd> write_color(), with gamma correction]
+
+
+<div class='together'>
+If we take the gamut now we get a much more consistent ramp from darkness to lightness:
```
Comment (hollasch) [inline books/RayTracingInOneWeekend.html:2545]: "If we take the gamut ..." → "Using this gamma correction, ..."

File: books/RayTracingInOneWeekend.html [hunk 4]
```diff
@@ -2510,6 +2447,107 @@
...
+
+  ![<span class='num'>Image 11:</span> The gamut of our renderer so far
```
Comment (hollasch) [inline books/RayTracingInOneWeekend.html:2547]: _gamut_: range or extent. That's not what we mean here. Perhaps change the caption to "A/The gamma-corrected rendering".

**Comment by trevordblack** [comment]
I followed all of your changes pretty much exactly

File: books/RayTracingInOneWeekend.html [hunk 5]
```diff
@@ -2542,9 +2542,9 @@
 
 
 <div class='together'>
-If we take the gamut now we get a much more consistent ramp from darkness to lightness:
+Using this gamma correction, we now get a much more consistent ramp from darkness to lightness:
 
-  ![<span class='num'>Image 11:</span> The gamut of our renderer so far
+  ![<span class='num'>Image 11:</span> The gamma-corrected of two diffuse spheres
```
Comment (hollasch) [inline books/RayTracingInOneWeekend.html]: "gamma-corrected <render/image> of two diffuse spheres"

or just

"The gamma-corrected rendering/result"

**Review by hollasch — APPROVED** [review]
One tiny error to fix.

File: books/RayTracingInOneWeekend.html (hunk 5 above)
Comment (trevordblack) [reply books/RayTracingInOneWeekend.html]: I went with render just before you sent out this comment. ha

**Pull request merged**